test/test_torch.py

```diff
@@ -13415,6 +13415,26 @@ def test_cat_big(self, device):
         result = torch.cat(concat_list)
         self.assertEqual(result.size(0), SIZE1 + SIZE2)
 
+    def test_index_add(self, device):
+        # test coverage for issue with atomic add:
+        # https://github.com/pytorch/pytorch/issues/29475
+        def is_signed(dtype):
+            return torch.is_signed(torch.tensor(0, dtype=dtype))
+        for dtype in torch.testing.get_all_math_dtypes(device):
+            # print(dtype)
```

Comment:
Forgot to delete comment?. One could toss the `dtype` into the self.assertEquals message if we really wanted to know what we're testing.

Reply:
I'll remove it, but I do sometimes like leaving easy-to-uncomment prints in tests sometimes, especially when constructing the info to print is more complicated.
In this case I don't always remember the order of the dtypes, so in development it's interesting to see which dtypes passed before a failure. changing just the assert doesn't fix that.

Reply:
In that case, the solution to the problem should be to add one test per dtype